mint.json

```diff
@@ -321,7 +321,8 @@
         "guides/FCM-iOS-Novu/how-to-send-PUSH-notifications-to-iOS-devices-with-FCM-using-Novu",
         "guides/fcm-flutter-novu/how-to-send-push-notifications-to-flutter-apps-with-fcm-using-Novu",
         "guides/how-to-integrate-segment-with-novu",
-        "guides/novu-fcm-react-native-android"
+        "guides/novu-fcm-react-native-android",
+        "guides/echo-react-email/echo-react-email"
```

Comment:
I see that you put the guide inside a `echo-react-email` directory. Is that necessary? 

If it needs to be in a folder, can it be in a `echo` folder so that all the guides that have to deal with echo can be inside the directory?